Makefile

```diff
@@ -53,6 +53,12 @@ srpm: tarball
 packages: rpm srpm
 
 
+# Containers
+images:
+	podman build -t tmt:mini -f ./containers/mini .
+	podman build -t tmt:full -f ./containers/full .
```

Comment:
I'd use `--squash` as well (although it should be default AFAIR). It doesn't squash base layers, just the newly created ones.

Reply:
Ok, added.